apps/addons/templates/addons/impala/details.html

```diff
@@ -92,7 +92,7 @@
 
 {# All this depends on the addon or version, and nothing needs the user,
    so we can cache it all against the addon. #}
-<section class="primary">
+<section class="primary" id="addon-description-header">
```

Comment:
The id is used for selecting the element here: https://github.com/mozilla/olympia/pull/1078/files#diff-7a64c91370d5382dd38f27dd1e7b09f2R5